js/packages/binderhub-client/lib/index.js

```diff
@@ -14,8 +26,15 @@ export class BinderRepository {
    * @param {URL} buildEndpointUrl API URL of the build endpoint to talk to
    * @param {string} [buildToken] Optional JWT based build token if this binderhub installation requires using build tokens
    * @param {boolean} [buildOnly] Opt out of launching built image by default by passing `build_only` param
+   * @param {string} [apiToken] Optional Bearer token for authenticating requests
    */
-  constructor(providerSpec, buildEndpointUrl, buildToken, buildOnly) {
+  constructor(
+    providerSpec,
+    buildEndpointUrl,
+    buildToken,
+    buildOnly,
+    { apiToken },
```

Comment:
Start accepting keyword arguments here, which we should probably do going forward, rather than continuing to add positional arguments. I'm not sure what our API stance is on changing this, otherwise I would have collapsed buildToken and buildOnly into the `options` dict, but that's a breaking API change here.

Reply:
@minrk agree re: keyword arguments. https://github.com/yuvipanda/jupyterhub-fancy-profiles is the only current user of this external to this repo, so I think you should just make the breaking change right now.